zerver/webhooks/reviewboard/view.py

```diff
@@ -175,14 +175,16 @@ def get_review_request_repo_title(payload: Dict[str, Any]) -> str:
 ALL_EVENT_TYPES = list(RB_MESSAGE_FUNCTIONS.keys())
 
 
-@webhook_view("ReviewBoard", all_event_types=ALL_EVENT_TYPES)
+@webhook_view("Review Board", all_event_types=ALL_EVENT_TYPES)
```

Comment:
I don't think we expect spaces here, since we construct a client name with these that is clearly meant to not have spaces:

```
def full_webhook_client_name(raw_client_name: Optional[str] = None) -> Optional[str]:                       
    if raw_client_name is None:                                                                             
        return None                                                                                         
    return f"Zulip{raw_client_name}Webhook"                                                                 
```

One option would be to change that function to remove internal spaces, but we should do something.

Reply:
Removed this hunk.